routes/web.php

```diff
@@ -225,18 +240,18 @@ function () {
             $qrcode = gzencode($qrcode, 9);
         }
         echo $qrcode;
-        
+
         //echo $qrcode->render($data);
-        
+
     }
```

Comment:
Function closing brace must go on the next line following the body; found 1 blank lines before brace